utilities/generate-kcp-api.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+
+set -e
+
+if ! which kubectl-kcp; then
+  echo "kubectl-kcp required on path"
+  echo "you can install it running:"
+  echo "    $ git clone https://github.com/kcp-dev/kcp && cd kcp && make install"
+  exit 1
+fi
+
+THIS_DIR="$(dirname "$(realpath "$0")")"
+CRD_DIR="$(realpath ${THIS_DIR}/../backend-shared/config/crd/bases)"
```

Comment:
I suppose we should generate these resources for other appstudio CRDs as well. `appstudio-shared/config/crd`